ports/stm32/omv_csi.c

```diff
@@ -192,6 +192,13 @@ static int stm_csi_config(omv_csi_t *csi, omv_csi_config_t config) {
         DCMI->CR &= ~(DCMI_CR_JPEG_Msk << DCMI_CR_JPEG_Pos);
         DCMI->CR |= (csi->pixformat == PIXFORMAT_JPEG) ? DCMI_JPEG_ENABLE : DCMI_JPEG_DISABLE;
         #else
+
+        // Reset pipe state to allow reconfiguring them. Note that
+        // abort() doesn't reset the state unless the pipe is active.
+        for (size_t i=0; i<DCMIPP_NUM_OF_PIPES; i++) {
+            csi->dcmi.PipeState[i] = HAL_DCMIPP_PIPE_STATE_RESET;
+        }
```

Comment:
This change seems to make set_pixformat() work only once per boot of the camera. Like, if I run a script, stop it, and then run another script then set_pixformat fails with an error until the system is power cycled.